src/include/duckdb/common/enums/join_type.hpp

```diff
@@ -27,10 +27,11 @@ enum class JoinType : uint8_t {
 	             // partner (false)
 	SINGLE = 8,  // SINGLE join is like LEFT OUTER JOIN, BUT returns at most one join partner per entry on the LEFT side
 	             // (and NULL if no partner is found)
-	RIGHT_SEMI = 9, // RIGHT SEMI join is created by the optimizer when the children of a semi join need to be switched
-	                // so that the build side can be the smaller table
-	RIGHT_ANTI = 10 // RIGHT ANTI join is created by the optimizer when the children of an anti join need to be
-	                // switched so that the build side can be the smaller table
+	RIGHT_SEMI = 9,  // RIGHT SEMI join is created by the optimizer when the children of a semi join need to be switched
+	                 // so that the build side can be the smaller table
+	RIGHT_ANTI = 10, // RIGHT ANTI join is created by the optimizer when the children of an anti join need to be
+	                 // switched so that the build side can be the smaller table
+	CROSS = 11
```

Comment:
Cross products are not handled as joins in DuckDB. Can we remove this from the internal enum again? We need to create a `CrossProductRelation` to create a cross product.

Reply:
Sure, will remove and come back